packages/docs-reanimated/src/components/InteractivePlayground/useEnteringExitingAnimationPlayground/Example.tsx

```diff
@@ -0,0 +1,270 @@
+  BounceIn: BounceIn,
+  BounceInRight: BounceInRight,
+  BounceInLeft: BounceInLeft,
+  BounceInUp: BounceInUp,
+  BounceInDown: BounceInDown,
+
+  // Flip
+  FlipInEasyX: FlipInEasyX,
+  FlipInEasyY: FlipInEasyY,
+  FlipInXDown: FlipInXDown,
+  FlipInXUp: FlipInXUp,
+  FlipInYLeft: FlipInYLeft,
+  FlipInYRight: FlipInYRight,
+
+  // LightSpeed
+  LightSpeedInRight: LightSpeedInRight,
+  LightSpeedInLeft: LightSpeedInLeft,
+
+  // Pinwheel
+  PinwheelIn: PinwheelIn,
+
+  // Roll
+  RollInRight: RollInRight,
+  RollInLeft: RollInLeft,
+
+  // Rotate
+  RotateInDownLeft: RotateInDownLeft,
+  RotateInDownRight: RotateInDownRight,
+  RotateInUpLeft: RotateInUpLeft,
+  RotateInUpRight: RotateInUpRight,
+
+  // Slide
+  SlideInRight: SlideInRight,
+  SlideInLeft: SlideInLeft,
+  SlideInUp: SlideInUp,
+  SlideInDown: SlideInDown,
+
+  // Stretch
+  StretchInX: StretchInX,
+  StretchInY: StretchInY,
+};
+
+export const EXITING_ANIMATIONS = {
+  // Fade
+  FadeOut: FadeOut,
+  FadeOutLeft: FadeOutLeft,
+  FadeOutRight: FadeOutRight,
+  FadeOutUp: FadeOutUp,
+  FadeOutDown: FadeOutDown,
+
+  // Bounce
+  BounceOut: BounceOut,
+  BounceOutRight: BounceOutRight,
+  BounceOutLeft: BounceOutLeft,
+  BounceOutUp: BounceOutUp,
+  BounceOutDown: BounceOutDown,
+
+  // Flip
+  FlipOutEasyX: FlipOutEasyX,
+  FlipOutEasyY: FlipOutEasyY,
+  FlipOutXDown: FlipOutXDown,
+  FlipOutXUp: FlipOutXUp,
+  FlipOutYLeft: FlipOutYLeft,
+  FlipOutYRight: FlipOutYRight,
+
+  // LightSpeed
+  LightSpeedOutRight: LightSpeedOutRight,
+  LightSpeedOutLeft: LightSpeedOutLeft,
+
+  // Pinwheel
+  PinwheelOut: PinwheelOut,
+
+  // Roll
+  RollOutRight: RollOutRight,
+  RollOutLeft: RollOutLeft,
+
+  // Rotate
+  RotateOutDownLeft: RotateOutDownLeft,
+  RotateOutDownRight: RotateOutDownRight,
+  RotateOutUpLeft: RotateOutUpLeft,
+  RotateOutUpRight: RotateOutUpRight,
+
+  // Slide
+  SlideOutRight: SlideOutRight,
+  SlideOutLeft: SlideOutLeft,
+  SlideOutUp: SlideOutUp,
+  SlideOutDown: SlideOutDown,
+
+  // Stretch
+  StretchOutX: StretchOutX,
+  StretchOutY: StretchOutY,
+};
+
+export default function Example({ entering, exiting }: ExampleProps) {
+  const [visible, setVisible] = React.useState(true);
+
+  const getEnteringAnimation = () => {
+    const {
+      animation,
+      easing,
+      duration,
+      delay,
+      isSpringBased,
+      mass,
+      stiffness,
+      damping,
+      overshootClamping,
+      restSpeedThreshold,
+      restDisplacementThreshold,
+    } = entering;
+
+    return !isSpringBased
+      ? ENTERING_ANIMATIONS[animation]
+          .duration(duration)
+          .delay(delay)
+          .easing(easing)
+      : ENTERING_ANIMATIONS[animation]
+          .delay(delay)
+          .springify()
+          .damping(damping)
+          .mass(mass)
+          .overshootClamping(overshootClamping)
+          .restSpeedThreshold(restSpeedThreshold)
+          .restDisplacementThreshold(restDisplacementThreshold)
+          .stiffness(stiffness);
```

Comment:
I don't see any reason to use inverse condition here.

```suggestion
    return isSpringBased
      ? ENTERING_ANIMATIONS[animation]
          .delay(delay)
          .springify()
          .damping(damping)
          .mass(mass)
          .overshootClamping(overshootClamping)
          .restSpeedThreshold(restSpeedThreshold)
          .restDisplacementThreshold(restDisplacementThreshold)
          .stiffness(stiffness)
        : ENTERING_ANIMATIONS[animation]
          .duration(duration)
          .delay(delay)
          .easing(easing);
```